.github/workflows/code-lint-and-prettify.yml

```diff
@@ -35,19 +93,22 @@ jobs:
           VALIDATE_YAML_PRETTIER: true
           VALIDATE_ENV: true
           VALIDATE_JAVASCRIPT_PRETTIER: true
-          FIX_JAVASCRIPT_PRETTIER: true
+          FIX_JAVASCRIPT_PRETTIER: false
```

Comment:
We should be able to run JavaScript prettier, you needed to ignore the lock file for prettier and not to disable it